internal/otelcollector/config/metric/agent/set_instrumention_scope.go

```diff
@@ -0,0 +1,31 @@
+package agent
+
+import (
+	"fmt"
+
+	"github.com/kyma-project/telemetry-manager/internal/otelcollector/config"
+	"github.com/kyma-project/telemetry-manager/internal/otelcollector/config/metric"
+)
+
+func transformInstrumentationScope(inputSource metric.InputSourceType) *TransformProcessor {
```

Comment:
This func does not transform anything
```suggestion
func makeInstrumentationScopeProcessor(inputSource metric.InputSourceType) *TransformProcessor {
```